spec/views/events/show.html.erb_spec.rb

```diff
@@ -64,6 +64,26 @@
     expect(rendered).to have_link(t(:print_button_label, scope: 'events.badges'))
   end
 
+  it "should not display accept-all-button for non-organizers" do
+    sign_in(FactoryGirl.create(:user, role: :coach))
+    render
+    expect(rendered).to_not have_link(I18n.t('events.applicants_overview.accept_all'))
+  end
+
+  it "should display accept-all-button for organizers if there are enough free places" do
+    sign_in(FactoryGirl.create(:user, role: :organizer))
+    @event.max_participants = Float::INFINITY
+    render
+    expect(rendered).to have_link(I18n.t('events.applicants_overview.accept_all'))
+  end
+
+  it "should not display accept-all-button if there are not enough free places" do
+    sign_in(FactoryGirl.create(:user, role: :organizer))
+    @event.max_participants = 0
```

Comment:
This is probably not the best test data, since @querenker mentioned in another discussion that events with 0 max_participants are not valid